### PR TITLE
Fix some typos in documentation

### PR DESCRIPTION
**Pull request by Calinou**

### CODE REVIEW

File: doc/01_index.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ Some general hints:
 
 * No matter what's the current keyboard layout or on which platform the
   game is running, the console is always opened and closed by pressing
-  **Left-Shift + Escape**.
-* Yamagi Quake II saves it's configuration, savegames and screenshots
+  <kbd>Left Shift + Escape</kbd>.
```
Comment (DanielGibson) [inline doc/01_index.md]: I don't think using `<kbd>` is really an improvement here (and it looks worse when viewing the file as a plain text file)

File: doc/06_contributing.md [hunk 2]
```diff
@@ -12,20 +12,20 @@ into the idea.
 
 Some rules to follow:
 
-* Use Github! Sign up for an account, fork our repository and send a
+* Use GitHub! Sign up for an account, fork our repository and send a
   pull request after you're satisfied with your work. We won't accept
-  patches send by mail or - even worse - as pastebin links.
+  patches sent by email or - even worse - as pastebin links.
```
Comment (DanielGibson) [inline doc/06_contributing.md]: email? shouldn't it be e-mail or E-Mail or something?

**Comment by DanielGibson** [comment]
Apart from my minor nitpicks this looks great, thanks for proof-reading! :)

File: doc/06_contributing.md (hunk 2 above)
Comment (Calinou) [reply doc/06_contributing.md]: [English seems to use email more often than e-mail](https://grammarist.com/style/e-mail-email/), so I opted for that form.

File: doc/01_index.md (hunk 1 above)
Comment (Calinou) [reply doc/01_index.md]: I reverted to bold instead of using `<kbd>`.

**Comment by DanielGibson** [comment]
Thanks!

**Pull request merged**